.circleci/windows/package.cfg

```diff
@@ -0,0 +1,79 @@
+[bundle]
+
+packages = {pkg_prefix}libsodium
+
+filename_stub = aeternity-windows
+
+display_name = Aeternity for Windows
+
+description = The full Aeternity node for Windows.
+
+version = VERSION_PLACEHOLDER
```

Comment:
This will be replaced by the node version automatically, to avoid changing the script with every release